src/stories/packages/components/ButtonOutline.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { ButtonOutline } from '@terra-ui-packages/components';
+import { ThemeProvider } from '@terra-ui-packages/components';
+
+import React from 'react';
+
+import { defaultBrand } from 'src/libs/brands';
```

Comment:
@nawatts do you have a suggestion on how to handle the branding? I would like these stories to NOT depend on things outside of `@terra-ui-packages/components`.

Reply:
Originally, I wanted to have the default Terra brand be the default value for the theme context. I think it would still make sense to at least put the default brand configuration in the components package.

I wonder if there's some Storybook configuration to wrap around all stories, so we could have a theme provider in one place instead of in every story.

Reply:
There has to be-- let me look into that.

Reply:
@nawatts I did this in https://github.com/DataBiosphere/terra-ui/pull/4580/commits/1998ae0ca5b10465858aab422af59bc53fc9d393 by adding a decorator to each Meta definition. This at least allowed me to isolate the `ThemeProvider` dependencies in `StoryThemeProvider`. And it allowed me to remove the custom render functions for the stories, so they are cleaner.

You can also put a [global decorator](https://storybook.js.org/docs/writing-stories/decorators#global-decorators) in `.storybook/preview.ts`, but the problem with that is it doesn't appear that the location of `preview.ts` is configurable, and since it is outside of the `src` directory, it can't refer to code in `src` or `@terra-ui-packages`.

Reply:
OK, I will give it a try again.

Reply:
Yes, that worked! I'm not sure what I did wrong last time… but now I just have the global decorator.